### ISSUE TITLE
Extensions (Watch app) isn't signed

### ISSUE THREAD

**Issue by SebastianThiebaud** [nanonation/floatsign#8]
Hello,

I use floatsign.sh for several months now and no problem so far. I just created a new Apple Watch extension on my project and now the app cannot be installed on my phone. Here's the iPhone logs:

```
May 27 17:51:34 Sebastiens-iPhone-6 installd[7377] <Error>: 0x100484000 -[MICodeSigningVerifier performValidationWithError:]: 188: Failed to verify code signature of <MIPluginKitPluginBundle : path = /private/var/mobile/Library/Caches/com.apple.mobile.installd.staging/temp.8czzP3/extracted/Payload/EasyShift.app/PlugIns/EasyShift WatchKit Extension.appex identifier = com.wise-view.easyshift-dev.watchkitextension type = 7> : 0xe800801c (Code not signed)
May 27 17:51:34 Sebastiens-iPhone-6 installd[7377] <Error>: 0x100484000 -[MIInstaller performInstallationWithError:]: Verification stage failed
May 27 17:51:34 Sebastiens-iPhone-6 itunesstored[7057] <Error>: 0x101f90000 __MobileInstallationInstallForLaunchServices_block_invoke240: Returned error Error Domain=MIInstallerErrorDomain Code=13 "Failed to verify code signature of <MIPluginKitPluginBundle : path = /private/var/mobile/Library/Caches/com.apple.mobile.installd.staging/temp.8czzP3/extracted/Payload/EasyShift.app/PlugIns/EasyShift WatchKit Extension.appex identifier = com.wise-view.easyshift-dev.watchkitextension type = 7> : 0xe800801c (Code not signed)" UserInfo=0x1555ab360 {LibMISErrorNumber=-402620388, LegacyErrorString=ApplicationVerificationFailed, SourceFileLine=188, FunctionName=-[MICodeSigningVerifier performValidationWithError:], NSLocalizedDescription=Failed to verify code signature of <MIPluginKitPluginBundle : path = /private/var/mobile/Library/Caches/com.apple.mobile.installd.staging/temp.8czzP3/extracted/Payload/EasyShift.app/PlugIns/EasyShift WatchKit Extension.appex identifier = com.wise-view.easyshift-dev.watchkitextension type = 7> : 0xe800801c (Code not signed)}
May 27 17:51:34 Sebastiens-iPhone-6 itunesstored[7057] <Warning>: ERROR: MobileInstallationInstallForLaunchServices returned nil
May 27 17:51:34 Sebastiens-iPhone-6 lsd[87] <Warning>: LaunchServices: installation failed for app com.wise-view.easyshift-dev
```

It looks like the extension isn't signed. Is there an option I'm missing or is floatsign.sh not supporting that yet? Thanks.

**Comment by SebastianThiebaud** [comment]
I just submitted a pull request that fixes this issue: see #9

**Comment by GoannaGuy** [comment]
My apologies that we've lagged so far behind on processing pull requests. I believe we've got all of them merged now.
